server/src/main/scala/typedapi/server/RouteExtractor.scala

```diff
@@ -48,7 +48,7 @@ trait RouteExtractorLowPrio {
 
     def apply(request: EndpointRequest, extractedHeaderKeys: Set[String], inAgg: EIn): Extract[Out] = request.uri match {
       case p :: tail => 
-        if (p == wit.value.toString()) 
```

Comment:
Why did you remove the braces exclusively here?

Reply:
Emacs is adding braces when interacting with Java methods and I never bothered to change that :). Therefore, you will find `toString()` multiple times in this code base.

Reply:
IntelliJ marks expressions with unnecessary braces yellow. And I dont like warnings ;). But in this case they will actually become necessary with dotty I think. This makes it possible to refer to the function itself without any extra `_`